fendermint/vm/actor_interface/src/ipc.rs

```diff
@@ -88,7 +84,7 @@ lazy_static! {
                             abi: ia::subnet_actor_manager_facet::SUBNETACTORMANAGERFACET_ABI
                                 .to_owned(),
                         },
-                        EthFacet {
+EthFacet {
```

Comment:
Strange that cargo fmt is not complaining here?

Reply:
I think it's because we're inside a macro, `lazy_static!`. Within these macros we don't get any formatting, which is why the indent is inconsistent [here](https://github.com/consensus-shipyard/ipc/blob/c0bec4e3bee187fe78a6811ef985333927d0aba7/fendermint/app/src/cmd/key.rs#L49-L73).